src/main/webapp/js/user-page-loader.js

```diff
@@ -34,37 +34,37 @@ function setPageTitle() {
  */
```

Comment:
I was wondering, should we have the ```user-page-loader.js``` follow this format as well?
```
(function(){
   // ... code here ...
})();
```
I guess my question is that should we always have our JavaScript files look like this because it prevents the possibility of having variables global? Also, shouldn't we have it where we fetch our messages first then to build the message div? It seems to me, as of right now, the messages are in a race condition?